.github/workflows/build.yml

```diff
@@ -207,79 +207,82 @@ jobs:
-
-          #     - if: |
-          #         (github.event_name == 'release' ||
-          #           (github.event_name == 'workflow_dispatch' &&
-          #            github.event.inputs.publish == 'true'))
-          #       name: Publish python package
-          #       env:
-          #         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          #         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-          #       run: |
-          #         twine upload --skip-existing wrappers/python/dist/*
+        run: npx lerna publish from-package --no-push --no-private --yes --no-git-tag-version
+
+  build-py:
+    name: Build Python
+    needs: [build-release]
+
+    strategy:
+      matrix:
+        include:
+          - runner: buildjet-2vcpu-ubuntu-2204
+            os: ubuntu-x86_64
+            library: library-linux-x86_64
+            plat-name: manylinux2014_x86_64
+          - runner: windows-latest
+            os: windows-x86_64
+            library: library-windows-x86_64
+            plat-name: win_amd64
+          - runner: macos-latest
+            os: macos-x86_64
+            library: library-darwin-x86_64
+            plat-name: macosx_10_9_universal2 # macosx_10_9_x86_64
+
+    runs-on: ${{ matrix.runner }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine auditwheel
+      - name: Fetch library artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{matrix.library}}
+          path: wrappers/python/anoncreds/
+
+      - name: Build python package
+        run: |
+          python setup.py bdist_wheel --python-tag=py3 --plat-name=${{ matrix.plat-name }}
+        working-directory: wrappers/python
+
+      - name: Test python package
+        shell: sh
+        run: |
+          cd wrappers/python
+          pip install --upgrade pip
+          pip install dist/*
+          python -m demo.test
+      - if: "runner.os == 'Linux'"
+        name: Auditwheel
+        run: auditwheel show wrappers/python/dist/*
+
+      - name: Upload python package
+        uses: actions/upload-artifact@v3
+        with:
+          name: python-${{ runner.os }}
+          path: wrappers/python/dist/*
+
+      - if: |
+          (github.event_name == 'release' ||
+            (github.event_name == 'workflow_dispatch' &&
+              github.event.inputs.publish == 'true'))
+        name: Publish python package
+      - name: Publish python package
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        run: |
+          twine upload --repository testpypi --skip-existing wrappers/python/dist/*
```

Comment:
For now this still releases on testpypi. If we did one release with this repository and it works, I will remove the `--repository testpypi` and add the correct, non-test, secret.